client/landing/stepper/hooks/use-track-scroll-to-bottom.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useRef } from 'react';
+
+type OptionalProps = {
+	[ key: string ]: unknown;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+
+const useTrackScrollToBottom = ( enabled: boolean, onEnd: any, optionalProps?: OptionalProps ) => {
```

Comment:
We also have a hook in stepper named `useTrackScrollPageFromTop`. Would be nice to keep naming consistent and name this `useTrackScrollPageToBottom` 🙂

Reply:
[nit] We can also use `onEnd: () => void` instead of `any`.